CHANGELOG.rst

```diff
@@ -3,6 +3,18 @@
 Unreleased
 ==========
 
+.. _changelog-4.6.0:
+
+4.6.0 — 2023-07-17
+==================
+
+🚀 Feature
+-----------
+
+- Websocket based dialect have been stabilized and is officially supported now
+
+    📘 Note: Inserting multiple empty row's, facilitating default settings currently isn't supported.
```

Comment:
```suggestion
    📘 Note: Inserting multiple empty row's, facilitating default settings currently isn't supported by Exasol.
```

Reply:
or is this specific to the websocket implementation

Reply:
I would say it it specific to the dialect. Trubodbc does not work Pyodbc "supports it". For the websocket protocoll we need extra code to support it with other primitives.